tests/conftest.py

```diff
@@ -1,4 +1,3 @@
-"""Configuration for Pytest."""
```

Comment:
```suggestion
"""Configuration for Pytest."""
```

This is meant to be here.